pyingest/parsers/ads_nameparser.py

```diff
@@ -0,0 +1,191 @@
+import sys
+import re
+from namedentities import named_entities
+
+try:
+    import ads.ADSCachedExports as ads_ex
```

Comment:
There are cases where the arxiv data have HTML entities rather than straight unicode.  I was able to remove one of the ent2u calls without raising the error rate by too much (up to around 0.3%), but I have to find a workaround for another because we also use semicolons for name separation, and I have to figure out how to "use semicolons that aren't part of HTML entities" instead.